packages/components/filters/filter-menu/src/trigger-button/trigger-button.tsx

```diff
@@ -0,0 +1,216 @@
+import { CaretDownIcon, CloseIcon } from '@commercetools-uikit/icons';
+import { Badge } from '../badge';
+import { Chip } from '../chip';
+import type { TAppliedFilterValue } from '../filter-menu';
+import styled from '@emotion/styled';
+import { designTokens } from '@commercetools-uikit/design-system';
+import SecondaryIconButton from '@commercetools-uikit/secondary-icon-button';
+import {
+  forwardRef,
+  LegacyRef,
+  RefObject,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+
+const { colorNeutral85, colorNeutral40, colorPrimary, fontSize20 } =
+  designTokens;
+
+const useScrollObserver = (ref: RefObject<HTMLElement>, totalCount: number) => {
```

Comment:
This is a really elegant solution to hiding `Chip`s and displaying a count of how many `Chip`s are hidden in the `Badge`.